pageserver/src/consumption_metrics.rs

```diff
@@ -19,6 +19,9 @@ use tracing::*;
 use utils::id::{NodeId, TenantId, TimelineId};
 
 const WRITTEN_SIZE: &str = "written_size";
+/// Values will be the difference of the latest written_size (last_record_lsn) to what we
+/// previously sent.
+const WRITTEN_SIZE_DELTA: &str = "written_size_delta";
```

Comment:
The metric we current expose to our partners and in our APIs ([see for example](https://api-docs.neon.tech/reference/listprojectsconsumption)) is called `written_data_bytes`. It's done via remapping event names from pageserver.

I wonder, can we call it `written_data_delta_bytes` right away to do not do any remapping (at least for now)?

Reply:
Yes let's change the name, that's the easiest of all :)

Reply:
[830a910](https://github.com/neondatabase/neon/pull/4822/commits/830a910033f17f4086b6ee8c0c305eecb99d4c7d)

Reply:
Another rename at 6e2c69697.